controller/service.go

```diff
@@ -3,17 +3,16 @@ package controller
 import (
 	"fmt"
 
+	"github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1"
+	"github.com/argoproj/argo-rollouts/utils/conditions"
+	logutil "github.com/argoproj/argo-rollouts/utils/log"
```

Comment:
Style nix: can we move this back below the external lib imports

Reply:
done